src/app/public/styles/_forms.scss

```diff
@@ -1,5 +1,12 @@
 @import "mixins";
 
+input[type=text].sky-form-control {
```

Comment:
@Blackbaud-AlexKingman Should we also do this for `[type=password]`, `[type=number]`, `[type=email]`, etc.? Basically any text input field...

Reply:
Not a bad idea, although its hard to tell what implications that would have. Maybe we could do so, but combine it with the specificity of `.sky-form-control`. That way its more deliberate. Sound like a good compromise?

I think we can just do this: `input[type].sky-form-control`.

Reply:
Hm, that would affect checkboxes...

Reply:
@Blackbaud-SteveBrush I looked further into this and `input[type]` seems like the right approach, with a caveat. This will apply the `appearance: none;` style to all inputs.

In my manual testing, everything still rendered as expected, and also fixed the issue with iOS inconsistencies. Here's the caveat: if someone were to create an input element WITHOUT any sky classes, it looks funky (based on your browser). However, this isn't a scenario we want, is it? Don't we always want our consumers to supply a `.sky-form-control` class? If so, I'd say this solution is the right move.